src/app/Instance/InstancesListPage/InstancesListPage.tsx

```diff
@@ -45,7 +45,7 @@ const InstancesListPage = (): JSX.Element => {
       accessor: "name",
       label: t("common.name"),
       formatter: (value: IRowData, row?: IRow): JSX.Element => {
-        const bridgeId = (row as Instance)?.id;
+        const bridgeId = (row as Instance)?.id ?? "";
```

Comment:
Just for my information to understand TS better; why was this needed? And, is it a good idea? IIUC `bridgeId` could have been a `string` or `undefined`. If undefined the `Link` would be broken whereas using `""` gives a link that isn't broken. However either way represents an invalid state (it should never happen). No change needed; I just wanted to understand the reason behind this change.

Reply:
@vpellegrino how can I simulate missing data? I am not able to delete them from the `data.ts` file.

Reply:
@manstis in the `BridgeResponse` interface, all fields are optional, so I made `Instance`'s field optional too.
Not a big issue, indeed. A broken link is properly handled in the detail page, by redirecting to the home (bridges list): https://github.com/vpellegrino/sandbox-ui/blob/6e3df9d8d9fa4086a427fcccc66fc897a559f866/src/app/Instance/InstancePage/InstancePage.tsx#L103

Moreover remember that this is a static check. Whenever API will return an empty string as a link, this is managed too.

Reply:
@sjamboro there (https://github.com/vpellegrino/sandbox-ui/blob/ef3de184493f3a4b4e540ab379a344d8f522114f/mocked-api/handlers.ts#L283) you can set empty array to fake empty processors.